code/game/atoms_movable.dm

```diff
@@ -251,24 +251,12 @@
 	return 1
 
 /atom/movable/Destroy(force)
-	var/inform_admins = (flags_2 & INFORM_ADMINS_ON_RELOCATE_2)
-	var/stationloving = (flags_2 & STATIONLOVING_2)
 
-	if(inform_admins && force)
-		var/turf/T = get_turf(src)
-		message_admins("[src] has been !!force deleted!! in [ADMIN_COORDJMP(T)].")
-		log_game("[src] has been !!force deleted!! in [COORD(T)].")
+	var/datum/component/stationloving/component = GetComponent(/datum/component/stationloving)

```

Comment:
Use COMSIG_PARENT_QDELETED, might need to modify it a bit to allow it to change the qdel hint